src/actions/userActions.spec.js

```diff
@@ -0,0 +1,107 @@
+import { expect } from 'chai';
+import * as userActions from '../actions/userActions';
+import * as types from '../actions/actionTypes';
+import thunk from 'redux-thunk';
+import nock from 'nock';
+import configureMockStore from 'redux-mock-store';
+import * as consts from '../constants/apiConstants.js';
+import { createStore } from 'redux';
+import initialState from '../reducers/initialState';
+import rootReducer from '../reducers';
+
+describe('User Actions', () => {
+  describe('Crear una cuenta de un usuario', () => {
+
+    it('devuelve la acción sendUserFormSuccess', () => {
+      const expectedAction = { type: types.SEND_USER_FORM_SUCCESS };
+      const action = userActions.sendUserFormSuccess();
+
+      expect(action).to.deep.equal(expectedAction);
+    });
+
+    it('devuelve la acción sendUserFormError', () => {
+      const errors = 'Error en el formulario';
+      const expectedAction = { type: types.SEND_USER_FORM_ERROR, errors: errors };
+      const action = userActions.sendUserFormError(errors);
+
+      expect(action).to.deep.equal(expectedAction);
+    });
+  });
+
+  const middleware = [thunk];
+  const mockStore = configureMockStore(middleware);
+
+  describe('Enviar el formulario de un usuario con acciones asíncronas', () => {
+    afterEach(() => {
+      nock.cleanAll();
+    });
+
+    describe ('Exitoso', () => {
+      const history = ["http://example/solicitud-registro"];
+        const user = {
+          name: 'Test One',
+          email: 'test@gmail.com',
+          phone: '099234567',
+          password: 'password',
+          password_confirmation: 'password'
+        };
+      it('Devuelve la acción sendUserFormSuccess al ejecutar sendUserForm', () => {
+        nock(consts.API_STAGING_URL)
+          .post('/users', user)
+          .reply(201);
+        const expectedAction = [{ type: types.SEND_USER_FORM_SUCCESS }];
+        const store = mockStore(initialState.userForm);
+        store.dispatch(userActions.sendUserForm(user, history)).then(() => {
+          expect(store.getActions()).to.deep.equal(expectedAction);
+        });
+      });
+
+      it('La acción sendUserFormSuccess cambia el estado de la store...NADA', () => {
+        nock(consts.API_STAGING_URL)
+          .post('/users', user)
+          .reply(201);
+
+        const action = {
+          type: types.SEND_USER_FORM_SUCCESS
+        };
+        const store = createStore(rootReducer, initialState);
+        store.dispatch(action);
+        expect(store.getState().user).to.not.equal({});
```

Comment:
Si decis q no cambia el store porque este test te pasa si comparas con `not equal`.
Ademas estas agarrando mal del store, porque tenes que comparar con `userForm` no con `user`